package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/avocado",
-  "version": "0.6.0",
+  "version": "0.6.1",
```

Comment:
no need bump version  for readme file change

Reply:
It locks the tslint version and need version bump.